api_tests/lib_sdk/cnd_instance.ts

```diff
@@ -80,10 +80,10 @@ export class CndInstance {
 
     public stop() {
         this.process.kill("SIGINT");
-        this.configFile = null; // Please see function is_running() below before changing this.
+        this.process = null;
     }
 
     public isRunning() {
-        return !this.configFile == null;
+        return !this.process == null;
```

Comment:
Is there some Typescript reason why this is not
```
return this.process != null;
```
??

Reply:
On looking up the order of precedence for Javascript this does not do what you want it to do I believe?  `!` has higher precedence that `==` so we need either
```
!(this.process == null)
```
or (preferably)
```
this.process != null
```